CMakeLists.txt

```diff
@@ -42,6 +42,21 @@ else()
    message(FATAL_ERROR "Git is required to build rogue!")
 endif()
 
+#####################################
+# Find python3
+#####################################
+set(DO_PYTHON 0)
+
+find_package(PythonInterp 3.6)
```

Comment:
I am assuming that there will be a python 3.7 in the future.  It looks like this check enforce version 3.6 only.  Do we want to be forward compatible with new python versions?

Reply:
This is a min version line. But these lines will have to be updated due to Ubuntu packaging styles:

````
# Next try Debian/Ubuntu suffix for boost
if ((NOT Boost_FOUND) AND DO_PYTHON)
   find_package(Boost 1.58 COMPONENTS system thread python-py36)
endif()

# Next try Mac with homebrew boost/python36
if ((NOT Boost_FOUND) AND DO_PYTHON)
   find_package(Boost 1.58 COMPONENTS system thread python36)
endif()
````